gen/gen.go

```diff
@@ -254,19 +255,25 @@ var SwaggerInfo = swaggerInfo{
 type s struct{}
 
 func (s *s) ReadDoc() string {
+	sInfo := SwaggerInfo
+	sInfo.Description = parseAndExecuteTemplate(sInfo.Description, sInfo)
```

Comment:
Read doc is called at run time. why do wen need  a template with func map called twice ?

Reply:
Because the `Description` includes template variables that will not be set today.
So the first template execution is just for the Description and the second is for the full Document.

Reply:
I was asking if you can't find a better way to solve this instead of calling the template 2 times.

Reply:
right, I see two ways;
- treat the Description as one template and the full document as one template and then apply the SwaggerInfo twice, like I did it, but I am definitely with you that it feels strange.
- apply the Description on the full document first and then apply the SwaggerInfo just once, the question would be have to do it best, something like `strings.Replace(doc, "{{.Description}}", sInfo.Description, -1)` could work

Reply:
if you do this before rendering template , i think it may be good enough . ( more eficient then parse+execute  the template 2 times). 
```
sInfo := SwaggerInfo
sInfo.Description = strings.Replace(doc, "{{.Description}}", sInfo.Description, 1)
```